flipperdevices/flipperzero-firmware
# WS: Show received signal age

Original feature by @LY2NEO with some fixes from me - [(read details)](https://github.com/DarkFlippers/unleashed-firmware/pull/197#issuecomment-1336348811)

Now rewritten to apply recommended UI changes, also contains extra change similar to #2111

# What's new

- When data is received from weather station it saves datastamp and calculates difference between actual time and data received time.
- Shows time counter on signal screen, when we reach above 1min it blinks with text [OLD], when we reach 59+ mins, it just shows [OLD]
- Now its possible to understand when signal was received if data on screen isn’t changed (for example temp stays the same)

# Verification 

- Launch Weather Station plugin, receive any supported temp sensor signal
- Select received signal and open it, check counter
- Send same signal again and see counter reset to 0

[Example_signal.zip](https://github.com/flipperdevices/flipperzero-firmware/files/10148576/Example_signal.zip)


# Checklist (For Reviewer)
